crates/shared/src/signature_validator/web3.rs

```diff
@@ -1,108 +0,0 @@
-use {
```

Comment:
This module had already been removed in #1842, but the code was accidentally left behind.